src/Console/ConsoleOptionParser.php

```diff
@@ -576,6 +576,7 @@ public function addSubcommand($name, array $options = [])
             $command = $name;
             $name = $command->name();
         } else {
+            $name = Inflector::underscore($name);
```

Comment:
Will `cake things initMyDb` still work? It looks like that will break which could cause a bunch of havoc for folks with cron and deploy automation.